yoda/api/key_entry/key_entry.h

```diff
@@ -171,7 +171,8 @@ struct Container<YT, KeyEntry<ENTRY>> {
   typedef KeyEntry<ENTRY> YET;
 
   // Event: The entry has been scanned from the stream.
-  void operator()(const ENTRY& entry) { map_[GetKey(entry)] = entry; }
+  // Save a copy! Stream provides copies of entries, that are desined to be `std::move()`-d away.
+  void operator()(ENTRY& entry) { map_[GetKey(entry)] = std::move(entry); }
```

Comment:
https://github.com/KnowSheet/Bricks/issues/169

This one is bigger than I thought.